app/components/search-results.js

```diff
@@ -45,21 +45,19 @@ export default Component.extend({
   },
 
   searchTask: task(function* (query) {
-    yield timeout(250);
-    const groupTypes = Object.keys(get(this, 'groups'));
-    groupTypes.forEach((type) => {
+    Object.keys(INDICES).forEach((type) => {
       get(this, `${type}Task`).perform(query).then((records) => {
         set(this, `groups.${type}`, records);
       }).catch((error) => {
         get(this, 'raven').captureException(error);
       });
     });
-  }).restartable(),
+    yield timeout(250);
```

Comment:
What's the reasoning behind the timeout here? The `.keepLatest()` modifier will drop any running task for a newly `.perform()`'d one

Reply:
`.keepLatest()` doesn't drop a currently-running task (that's `.restartable()`), it just limits the queue to a single task instance, so this should rate-limit to every 250ms